vhosting/webstack/zendserver.sls

```diff
@@ -71,7 +71,7 @@ include:
     - onlyif: test -e /usr/local/zend/etc/conf.d/zray.ini
     - watch_in:
       - service: zendserver
-{%- if webserver == 'nginx' %}
-      - service: php-fpm
-{% endif -%}
+{%- for php_version in php_versions %}
+      - service: php{{ php_version }}-fpm
```

Comment:
Deze change is niet backwards compatible.... ik zou deze eerst laten testen in de praktijk!

Reply:
volgens mij wel, want volgens de formula wordt php-fpm (zonder versie) alleen geinstalleerd als php_versions|length == 0 and webserver_edition == 'vanilla', dus niet zendserver. Die 'if' die ik heb weggehaald gaat dus nooit goed. In mijn versie worden gewoon alle php{versie}-fpm services herstart. Ik zie geen flow waarin dat niet goed zal gaan.